### ISSUE TITLE
POE Lookout - Base Kanban page 

### ISSUE THREAD

**Issue by tjwdev** [US-CBP/GTAS-UI#730]
The base POE lookout page has the ability to:
1. Fetch and load user generated lanes from the database
2. Fetch, construct load and assign tiles of data to their respective lanes
3. Allow manual updating of those statuses by dragging tiles into specific lanes
4. Allow automatic updating of tile statuses via timeout values for either missing the flight landing/taking off, or by having the update status being inactive.

**Comment by tjwdev** [comment]
For this state of the POE Lookout there are a few work flows to explain.

1. What are the ways a Passenger can be "Promoted To Lookout"?
- Through automatic promotion via hit category attribute
- Through manual promotion via the vetting page's PVL management, or on the Passenger Detail Page via Manual Hit

2. What is the automatic promotion?
- Hit categories now have a new boolean attribute that can be toggled active. 
- Under the hit categories management under the admin tab editing or creating a new hit category will allow you to select "Enable Automatic Lookout Promotion"
- Any rule or watchlist using this category will set the Passenger it hits upon to "ACTIVE" status on the POE Lookout page.
Note: This will not retroactively make "ACTIVE" any passengers previously hit, only future hits from the given rule with the applied category.

3. What is manual promotion?
- You may manually promote a passenger that has been hit and is not already lookout status "ACTIVE" by going to the vetting page and from the "Choose Action" drop down on a given row, select "Promote To Lookout" to set the given passenger's lookout status "ACTIVE".
Note: If a lookout status is not "NOTPROMOTED" or "DEMOTED", the option to promote the passenger will not be available.
- You may also promote a person without a hit by applying a Manual Hit on the Passenger Detail page of the given passenger, and selecting a category for the Manual Hit that is toggled with "Automatic Lookout Promotion".

4. There is an automatic inactivity flow on the lookout
- There are 3 types of automatic movement on the POE. 
- If the Lookout is not "NOTPROMOTED", "DEMOTED", "INACTIVE", or "MISSED" after a set amount of time of not having an update on the hitViewStatus it will automatically be moved to "INACTIVE"
- If the Lookout is "INACTIVE" for a certain amount of time, it will be automatically be moved to "DEMOTED"
- If the Lookout's flight has either departed or arrived (inbound/outbound depending) after a certain buffer period, it will be automatically be moved to "MISSED".
- If the Lookout status is updated AFTER it has been set to "MISSED", it will not be reset to "MISSED" automatically.

To summarize: 
<Any Non Inactive/NotPromoted/Demoted/Missed Status> --after no update time--> Inactive
<Any Non Inactive/NotPromoted/Demoted/Missed Status> --after flight missed buffer time--> Missed
Inactive --after inactivity time--> Demoted

These variables are located in the POEServiceImpl, they are called:
 inactiveWindowVariable, fullyDemoteVariable, missedFlightBufferVariable. 

Adjusting their date times will affect how quickly or slowly programmatic progression of the status' go.
